homeassistant/components/fritz/__init__.py

```diff
@@ -60,6 +64,30 @@ def _async_unload(event):
     return True
 
 
+async def async_device_setup(
+    hass: HomeAssistant, entry: ConfigEntry, fritz_tools: FritzBoxTools
+):
+    """Set up a device that is online."""
+    dev_reg = await device_registry.async_get_registry(hass)
```

Comment:
```suggestion
    dev_reg = device_registry.async_get(hass)
```

https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/device_registry.py#L642

Reply:
Since nothing is going to be `await`ed in the function, change this to `def async_device_setup` and decorate it with `@callback`

Reply:
Done